tests/src/Functional/PersonTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_starter_content\Functional;
+
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
+use Drupal\Tests\TestFileCreationTrait;
+
+/**
+ * Tests the Person content type.
+ *
+ * @group oe_starter_content
+ */
+class PersonTest extends BrowserTestBase {
```

Comment:
Tests for news and events are called *IntegrationTest.
I don't think they should, so I choose the simpler name here.

In oe_content, these tests would be in the respective modules.
For this test I followed the convention used here, so it is in the root /tests/ directory.